pkg/asset/machines/master.go

```diff
@@ -559,6 +559,15 @@ func (m *Master) Generate(ctx context.Context, dependencies asset.Parents) error
 			}
 			machineConfigs = append(machineConfigs, ignPowerSMT)
 		}
+
+		if installConfig.Config.Publish == types.InternalPublishingStrategy &&
+			(len(installConfig.Config.ImageDigestSources) > 0 || len(installConfig.Config.DeprecatedImageContentSources) > 0) {
+			ignChrony, err := machineconfig.ForCustomNTP("master", "time.adn.networklayer.com")
```

Comment:
Part of me would like if we didn't hard code this and stored it somewhere logical.. but I'm not sure where. Thoughts @r4f4 ?

Reply:
What about `pkg/types/powervs/defaults/`? Or maybe `pkg/constants/powervs/`?

Reply:
Thanks for the suggestion. I coded it up and force-pushed. PTAL.